src/components/App.tsx

```diff
@@ -1256,8 +1261,13 @@ class App extends React.Component<AppProps, AppState> {
 
         this.scheduleImageRefresh();
       },
+      THROTTLE_NEXT_RENDER && window.EXCALIDRAW_THROTTLE_RENDER === true,
```

Comment:
its not clear what value this is for so can you create a varibale `shouldThrottle` and assign it this ?

Reply:
IMO we should be using the inlay hints in vscode for this:
 
![image](https://user-images.githubusercontent.com/5153846/179253653-44cb59b4-d38a-4d74-ac0b-8c8dc9dc0797.png)

Putting it into a separate variable will make it confusing with respect to the `THROTTLE_NEXT_RENDER` variable.

Reply:
yeah in editor the hints show up but in this case it affects readability mostly coz of the naming and there are too many args so might wanna just pass a config object instead of individual params.
Also the variable `THROTTLE_NEXT_RENDER` could be renamed to `IS_RESIZING`.

Reply:
the variable `THROTTLE_NEXT_RENDER` indicates whether throttle should be disabled. While it is currently coupled to resizing, we should keep it clear what's going here (besides, it's just a subset of resizing behavior that we're concerned about)